config.yaml

```diff
@@ -43,7 +43,12 @@ siswrap_service_port: 10700
 
 # The service URL polled in the runfolder sensor:
 runfolder_svc_urls:
-  - http://testarteria1:10800
+  - http://testarteria1:10800/api/1.0/runfolders/pickup
+
+incoming_svc_urls:
+  - 
```

Comment:
Could this be on the same line, i.e.
```
- url: <...>
  dest_folder: <...>
```
I find that to be easier to read. Or does that not work with the yaml specification?